releasenotes/notes/2.1/extend-deprecation-tuple-inst-f24ab7a5632191e3.yaml

```diff
@@ -4,6 +4,6 @@ deprecations_circuits:
     The deprecated tuple-like interface for :class:`~.circuit.CircuitInstruction` was not
     removed in this release as originally planned. It will be removed in Qiskit 3.0.0
```

Comment:
```suggestion
    removed in this release as originally planned. It will be removed in Qiskit v3.0
```

Reply:
Here, 3.0.0 was used and above it's just 3.0.  Should that be consistent?

Reply:
Having consistency here would be nice. I've updated other places where it made sense but left a few mentions explicit (e.g. 2.3.0 and not v2.3) where the extra accuracy seems to be more appropriate.